src/khoj/processor/conversation/utils.py

```diff
@@ -97,23 +97,33 @@ def generate_chatml_messages_with_context(
     messages = user_chatml_message + rest_backnforths + system_chatml_message
 
     # Truncate oldest messages from conversation history until under max supported prompt size by model
+    messages = truncate_message(messages, max_prompt_size[model_name], model_name)
+
+    # Return message in chronological order
+    return messages[::-1]
+
+
+def truncate_message(messages, max_prompt_size, model_name):
```

Comment:
Nit: naming of `truncate_messages` function
```suggestion
def truncate_messages(messages, max_prompt_size, model_name):
```